app/components-react/hooks.ts

```diff
@@ -150,3 +150,16 @@ export function useForceUpdate() {
   const [, forceUpdate] = React.useReducer(x => x + 1, 0);
   return forceUpdate;
 }
+
+export function useInterval(callback: () => void, delay: number) {
```

Comment:
I think maybe we should rename this hook.  It forces a re-render every time the interval runs.  This is what you want here, and you're explicitly using that side effect.  But the name of the hook does not make it clear that's the case.

Reply:
maybe something like `useRenderInterval`?